mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
+disallow_untyped_calls = True
```

Comment:
I'm pretty sure `--strict` enables this by default (which we have in our `.pre-commit-config.yaml`. Do we need this?

Reply:
removed this as strict arg covers the config